test/cases/020_kernel/tags.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# common tags tests
+
+#!/bin/sh
+# SUMMARY: Test existence and correctness of kernel builder tag, label and file
+# LABELS:
+# REPEAT:
+
+set -e
+
+# Source libraries. Uncomment if needed/defined
+#. "${RT_LIB}"
+. "${RT_PROJECT_ROOT}/_lib/lib.sh"
+
+if [ -z "${KERNEL}" ]; then
+	echo "KERNEL env var must be set"
+	exit 1
+fi
+
+NAME=tags
+
+clean_up() {
+	docker rm ${ctrid}
+	/bin/rm -f ${BUILDERFILE}
+}
+trap clean_up EXIT
+
+# check the kernel images for tags, labels, files
+BUILDER=${KERNEL}-builder
+BUILDERFILE=/tmp/kernel-builder-${KERNEL}
```

Comment:
CI is failing because KERNEL is of the form `linuxkit/kernel:5.11.18` so this assumes a subdirectory of `/tmp/kernel-builder-linuxkit`. you might want to extract just the version number with `KERNEL_VERSION=$(echo $KERNEL | cut -d: -f 2)`, check that it's non-zero in length...or whatever your favourite string processing utility is ...

Reply:
Nice one. Thanks

Reply:
I am just going to make it a random file and be done with it.